cmd/verify_test.go

```diff
@@ -157,6 +159,7 @@ func TestCertify(t *testing.T) {
 
 		cmd.SetArgs([]string{
 			"-e", "is-helm-v3", // only consider a single check, perhaps more checks in the future
+			"-V", "4.9",
```

Comment:
The `openshift-version` is passed but not verified in the output report.

Reply:
Done. f3ae774f1f1304b39bdc942c7ad2bc304ad3ddf3